Marsiple/App/New Group/AlbumsViewController.swift

```diff
@@ -65,10 +64,12 @@ private extension AlbumsViewController {
         }
     }
     
-    func populatePhotoCollectionViewControllers() {
+    func populatePhotoCollectionViewControllers() -> [PhotoCollectionViewController] {
+        var viewControllers = [PhotoCollectionViewController]()
         albums.forEach {
```

Comment:
The preferred and cleaner way would be to use `map` to transform the albums into an array of `PhotoCollectionViewController` instances.

As so:
```
func populatePhotoCollectionViewControllers() -> [PhotoCollectionViewController] {
    return albums.map { PhotoCollectionViewController(photos: $0.photos) }
}
```

or even a bigger simplification:

```
private lazy var photoCollectionViewControllers = albums.map { PhotoCollectionViewController(photos: $0.photos) }
```

Reply:
Cool.